Javascript/q46.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>q46</title>
+
+  <style>
+    body {
+      font-family: Verdana, Geneva, Tahoma, sans-serif;
+    }
+
+    .flex {
+      display: flex;
+      gap: 10px;
+      flex-wrap: wrap;
+    }
+  </style>
+
+  <script>
+    function myFunction() {
+      const a = Number(document.querySelector("#demo0").value);
+      const b = Number(document.querySelector("#demo1").value);
+      const c = document.querySelector("#demo2").value;
+
+      switch(c){
+      case "+": document.getElementById("demo3").innerHTML= a+b;
```

Comment:
Please do proper indentation in this switch case block and take a look at the clean way of defining switch case blocks. It usually goes like this

```js
switch(c) {
  case '+':
    // Do something
    break;
}
```

Reply:
ok